wolfssl/wolfcrypt/pkcs7.h

```diff
@@ -48,7 +48,7 @@
 
 /* Max number of certificates that PKCS7 structure can parse */
 #ifndef MAX_PKCS7_CERTS
-    #define MAX_PKCS7_CERTS 4
+    #define MAX_PKCS7_CERTS 15
```

Comment:
Should this be low by default and made larger for specific builds?

Reply:
Agreed, that looks too big for a default.